.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Rspec should not be part of this PR, we will get there in a second! But for this moment, can you please remove it?